src/liballoc/range.rs

```diff
@@ -19,6 +16,7 @@ use Bound::{self, Excluded, Included, Unbounded};
 
 /// `RangeArgument` is implemented by Rust's built-in range types, produced
 /// by range syntax like `..`, `a..`, `..b` or `c..d`.
+#[stable(feature = "collections_range", since = "1.21.0")]
```

Comment:
Unless this is meant to be backport to beta, the `since` should be 1.22.0

Reply:
Woops! Fixed. thanks @kennytm!